man/udev.xml

```diff
@@ -68,9 +68,10 @@
       pointing to the device node, or run a specified program as part of
       the event handling.</para>
 
-      <para>A rule consists of a comma-separated list of one or more key-value pairs.
-      Each key has a distinct operation, depending on the used operator. Valid
-      operators are:</para>
+      <para>A rule consists of a comma-separated list of one or more key-operator-value pairs.
+      Each key has a distinct operation, depending on the used operator.</para>
```

Comment:
Maybe "Each such sequence has a distinct effect, depending on the key and operator used." ?